lib/node_modules/@stdlib/utils/is-capitalized/test/test.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var isCapitalized = require( './../lib' );
+
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.equal( typeof isCapitalized, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'the function returns `true` if provided a capitalized string', function test( t ) {
```

Comment:
"Capitalized string" has different connotations; e.g., all uppercase, each word having a capital letter, first letter is uppercase, etc. Instead, to be more explicit,

> the function returns `true` if provided a string having an uppercase first character